### PR TITLE
Make Quick optional

### PR DESCRIPTION
**Pull request by marcelofabri**
Continued work from #79

I've bumped the podspec to 6.0 because I guess this is a breaking change if people rely on it importing `Quick`.

// cc @coverback

### CODE REVIEW

File: Bootstrap/Podfile.lock [hunk 1]
```diff
@@ -35,10 +34,10 @@ CHECKOUT OPTIONS:
     :git: https://github.com/facebook/ios-snapshot-test-case.git
 
 SPEC CHECKSUMS:
-  FBSnapshotTestCase: 094f9f314decbabe373b87cc339bea235a63e07a
+  FBSnapshotTestCase: '094f9f314decbabe373b87cc339bea235a63e07a'
```
Comment (orta) [inline Bootstrap/Podfile.lock]: heh @ashfurrow

File: README.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 target 'YOUR_APP_NAME_HERE_Tests', :exclusive => true do
   pod 'Nimble-Snapshots'
+  pod 'Quick' # if you want to use it with Quick
```
Comment (orta) [inline README.md]: 👍

File: Bootstrap/Podfile.lock (hunk 1 above)
Comment (marcelofabri) [inline Bootstrap/Podfile.lock]: why only this entry is in quotes? 🤔

Comment (orta) [inline Bootstrap/Podfile.lock]: We talked about it this week:

screen shot 2017-05-26 at 11 23 56

Comment (marcelofabri) [inline Bootstrap/Podfile.lock]: but shouldn't all lines that I've touched be in quotes? 🕵️ 

anyway, using `ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-darwin16]` here 😂

Comment (coverback) [inline Bootstrap/Podfile.lock]: No, it only affects those that start with zero. And it's about `psych` gem version IIRC, not ruby itself.

Also, great job!

**Comment by ashfurrow** [comment]
Cool, thanks! Pushed this to CocoaPods + GitHub for Carthage.

**Pull request merged**